code/game/mecha/mecha.dm

```diff
@@ -1070,6 +1070,30 @@
 	return
 
 
+/obj/mecha/verb/checkSeat()

```

Comment:
You know this is literally not the solution, right?
At /tg/ we had an unused stub proc that was supposed to prevent dropping inside of atoms (the proc belonged to that atom), all we did was hook that up properly to the ```drop_item()``` ability.

Whether you have this proc or not (but if you do your laziness is criminal) you should do that instead.